packages/buckram/assets/styles/components/media/_images.scss

```diff
@@ -211,7 +211,6 @@
       box-sizing: border-box;
       float: $image-alignnone-caption-float;
       padding-left: if-map-get($image-alignnone-caption-left-padding, $type);
-      width: if-map-get($image-alignnone-caption-width, $type) !important;
```

Comment:
no width is neccesary for nonaligment class for images in prince, in order to set caption with full width by default